src/black/files.py

```diff
@@ -63,17 +63,18 @@ def find_project_root(
     if not srcs:
         srcs = [str(Path.cwd().resolve())]
 
-    path_srcs = [Path(Path.cwd(), src).resolve() for src in srcs]
+    src_parents = list()
+    for src in srcs:
```

Comment:
Not clear that this improves performance, and if it does, not clear the change is worth the cost in decreased readability.